### ISSUE TITLE
pip install fails with UnicodeDecodeError: 'gbk' codec can't decode byte 0xa5 in position 688: illegal multibyte sequence

### ISSUE THREAD

**Issue by tkf** [JuliaPy/pyjulia#218]
as @YP-Ye reported here https://github.com/JuliaPy/pyjulia/issues/198#issuecomment-435646505

@YP-Ye What is your `pip --version`?

**Comment by YP-Ye** [comment]
> as @YP-Ye reported here [#198 (comment)](https://github.com/JuliaPy/pyjulia/issues/198#issuecomment-435646505)
> 
> @YP-Ye What is your `pip --version`?

My pip version is 10.0.1. Do I need to upgrade pip?

**Comment by tkf** [comment]
Probably not. I was interested in the full output of `pip --version` which includes the Python version.

Are you using Python 2?

**Comment by YP-Ye** [comment]
> Probably not. I was interested in the full output of `pip --version` which includes the Python version.
> 
> Are you using Python 2?

I use python 3.6. The newest pip does not work.
C:\Windows\System32>pip --version
pip 18.1 from c:\anaconda3\lib\site-packages\pip (python 3.6)

**Comment by tkf** [comment]
I wrote a (possible) fix #219.  What happens if you try the following?

```sh
pip install --user 'https://github.com/tkf/pyjulia/archive/fix-setup-py.zip#egg=julia'
```

BTW, please quote code blocks https://discourse.julialang.org/t/psa-how-to-quote-code-with-backticks/7530
